pandas/_libs/tslibs/timedeltas.pyx

```diff
@@ -1910,9 +1932,16 @@ class Timedelta(_Timedelta):
 
             if other.dtype.kind == 'm':
                 # also timedelta-like
-                if self._reso != NPY_FR_ns:
-                    raise NotImplementedError
-                return _broadcast_floordiv_td64(self.value, other, _floordiv)
+                # TODO: could suppress
+                #  RuntimeWarning: invalid value encountered in floor_divide
```

Comment:
This would be great (especially if it comes from a pandas routine) so we can turn on -W:::pandas in the future